### PR TITLE
Fix tests for add puppet module to content view functionality

### LINKED ISSUE #3639: Fix add_puppet_module helper for content_view tests
Fix next downstream tests:
tests.foreman.ui.test_contentview.ContentViewTestCase.test_positive_add_puppet_module
tests.foreman.ui.test_contentview.ContentViewTestCase.test_positive_create_composite
tests.foreman.ui.test_contentview.ContentViewTestCase.test_negative_add_puppet_repo_to_composite

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #3639 
```
nosetests tests/foreman/ui/test_contentview.py -m test_positive_create_composite
.
----------------------------------------------------------------------
Ran 1 test in 585.155s

OK
```
```
nosetests tests/foreman/ui/test_contentview.py -m test_negative_add_puppet_repo_to_composite
.
----------------------------------------------------------------------
Ran 1 test in 50.143s

OK
```
```
nosetests tests/foreman/ui/test_contentview.py -m test_positive_add_puppet_module
.
----------------------------------------------------------------------
Ran 1 test in 86.272s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -540,11 +538,8 @@ def test_positive_create_composite(self):
             self.content_views.publish(cv_name2)
             self.content_views.create(composite_name, is_composite=True)
             session.nav.go_to_select_org(org.name)
-            session.nav.go_to_content_views()
             self.content_views.add_remove_cv(
                 composite_name, [cv_name1, cv_name2])
-            self.assertIsNotNone(self.content_views.wait_until_element(
```
Comment (oshtaier) [inline tests/foreman/ui/test_contentview.py]: 1) It is redundant verification as we have proper validation in `add_remove_cv` method itself
2) It is invalid verification as alert disappears when we switch between sub-tabs

File: tests/foreman/ui/test_contentview.py [hunk 2]
```diff
@@ -658,10 +651,10 @@ def test_negative_add_puppet_repo_to_composite(self):
             with self.assertRaises(UIError) as context:
                 self.content_views.add_puppet_module(
                     composite_name, 'httpd', filter_term='Latest')
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not find tab to add puppet_modules'
-                )
+            self.assertEqual(
```
Comment (oshtaier) [inline tests/foreman/ui/test_contentview.py]: Created https://github.com/SatelliteQE/robottelo/issues/3644 to track all similar changes

**Comment by abalakh** [comment]
ACK

**Comment by rochacbruno** [comment]
ACK

**Pull request merged**